src/api/api.types.tsx

```diff
@@ -250,10 +250,14 @@ export interface ImagePost {
   description?: string | null;
 }
 
-export interface Image
+export interface APIImage
   extends Required<Omit<ImagePost, 'upload_file'>>,
     CreatedModifiedMixin {
   id: string;
   primary: boolean;
   thumbnail_base64: string;
 }
+
+export interface ImageGet extends APIImage {
```

Comment:
The function name `Image` conflicts with the internal JavaScript constructor, which is why I renamed it to `APIImage`. I’m not inclined to use ImageMetadata because properties like `url `and `thumbnail_base64` are not strictly image metadata. They represent additional data related to the image but don’t align with the traditional definition of metadata for images. For now, I’ll stick with `APIImage` as it better reflects the broader scope of the object and its intended purpose.

Reply:
We are treating the metadata as id, file_name, primary and thumbnail. When we originally discussed it we were going to put the thumbnail inside the file metadata in the store, only we moved it to the database instead. The url is not in that schema in the first place. I didn't realise the name conflict `APIImage` sounds fine then.

Reply:
```
export interface APIImage
  extends Required<Omit<ImagePost, 'upload_file'>>,
    CreatedModifiedMixin {
  id: string;
  primary: boolean;
  thumbnail_base64: string;
}

export interface APIImageWithURL extends APIImage {
  url: string;
}
```

I've changed it to this as this is more descriptive. I also didn't like `ImageGet` 🤣

Reply:
Yeah I am happy with that. I would argue that APIImage implies its an image object and not its metadata but it should be clear enough when you hover over it.